config/environment.js

```diff
@@ -1,9 +1,10 @@
 /* eslint-env node */
+'use strict';
 
 module.exports = function(environment) {
-  var ENV = {
+  let ENV = {
     modulePrefix: 'code-corps-ember',
-    environment: environment,
+    environment,
```

Comment:
This file was changed due to running `ember init` and manually merging. Makes it more eslint compliant.